### DISCUSSION TITLE
Ubuntu 20?

### DISCUSSION THREAD

**Discussion by stevefxp1** [Security-Onion-Solutions/securityonion#5427] [Unsupported Versions] [answered]
Is there support for 20.04.03?

**Comment by TOoSmOotH** [comment, marked as answer]
Not at this time. There is a PR in that shows you what you would need if you want to manually try to make it run on 20.04. If someone wants to add the appropriate logic that is needed to support both 18 and 20 that would be very helpful.

https://github.com/Security-Onion-Solutions/securityonion/pull/5298
